.github/workflows/test.yml

```diff
@@ -0,0 +1,211 @@
+name: Run tests
+on: [push]
+
+jobs:
+  docs:
+    name: ${{ matrix.compose_app }}
+    strategy:
+      fail-fast: false
+      matrix:
+        compose_app:
+          - flake8
+          - docs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - run: git clone https://github.com/django/django.git --depth=1 ${GITHUB_WORKSPACE}/django
+      - run: docker-compose pull --include-deps ${{ matrix.compose_app }} || true
+      - run: DJANGO_PATH=${GITHUB_WORKSPACE}/django docker-compose build --pull ${{ matrix.compose_app }}
+      - run: DJANGO_PATH=${GITHUB_WORKSPACE}/django docker-compose run ${{ matrix.compose_app }}
+
+  sqlite:
+    name: sqlite
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version:
+          - 3.6
+          - 3.7
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - run: git clone https://github.com/django/django.git --depth=1 ${GITHUB_WORKSPACE}/django
+      - run: docker-compose pull --include-deps sqlite || true
+      - run: DJANGO_PATH=${GITHUB_WORKSPACE}/django docker-compose build --pull sqlite
+        env:
+          PYTHON_VERSION: ${{ matrix.python_version }}
+      - run: DJANGO_PATH=${GITHUB_WORKSPACE}/django docker-compose run sqlite
+        env:
+          PYTHON_VERSION: ${{ matrix.python_version }}
+
+  sqlite-gis:
+    name: sqlite-gis
+    strategy:
+      fail-fast: false
+      matrix:
+        python_version:
+          - 3.6
+          - 3.7
+    runs-on: ubuntu-latest
```

Comment:
instead of ubuntu latest wouldn't it better to use ubuntu LTS like ubuntu 20.04 and in future 22.04 etc